spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -202,31 +202,32 @@ The preceding output has three lines that of interest to us here:
 
 Now that we have a working task, we can intentionally create an error, to show how a Spring Cloud Task handles errors. To do so:
 
-. Open `src/main/java/io/spring/task/sample/timestamp/TimestampApplication.java` and insert the following line into the run method:
-+
+1) Open `src/main/java/io/spring/task/sample/timestamp/TimestampApplication.java` and insert the following line into the run method:
```

Comment:
Any reason why the switch from `.` which will auto-generate the bullets to explicit numbers?

Reply:
I see this in RC1 and it seems correctly numbered:
![image](https://user-images.githubusercontent.com/1562654/53496650-525e7a80-3a57-11e9-884d-10a4c23930b1.png)

Reply:
Yes, bad experience on mobile or small screen